DOC-2015: Added TINY-9735 to the release notes. (#2713)
tinymce/tinymce-docs: modules/ROOT/pages/6.5-release-notes.adoc

The following premium plugin updates were released alongside {productname} 6.5.

=== Advanced Code Editor 3.3.0

The {productname} 6.5 release includes an accompanying release of the **Advanced Code Editor** premium plugin.

**Advanced Code Editor** 3.3.0 includes the following fix.

> === When switching to full-screen mode when the editor was set to inline view, focus was not returned to the code editor on iOS devices.

Reviewer: ```suggestion
=== When switching to full-screen mode when the editor was set to inline view, focus was not returned to the code editor on iOS.
```

> === While switching to full-screen mode when the editor is set to inline view would not return the focus back to the code editor on iSO devices.

Reviewer: ```suggestion
=== While switching to full-screen mode when the editor is set to inline view would not return the focus back to the code editor on iOS devices.
```

> === While switching to full-screen mode when the editor is set to inline view would not return the focus back to the code editor on iOS devices.

Reviewer: ```suggestion
=== When switching to full-screen mode when the editor was set to inline view, focus was not returned to the code editor on iOS devices.
```

//#TINY-9735

> In previous versions of {productname}, the default behavior of `editor.execCommand` in inline mode on iOS was to hide the editor when the view is open. Although it shouldn't be possible to focus on it, iOS allowed it.

Reviewer: ```suggestion
In previous versions of {productname}, the default behavior of `editor.execCommand` in inline mode on iOS was to hide the editor when the view is open. Although it should not be possible to focus on it, iOS allowed it.
```

> As a consequence, when pressing the fullscreen button the editor used `editor.execCommand('mceFullScreen')`, which caused the cursor to be positioned incorrectly on iOS.

Reviewer: ```suggestion
As a consequence, when pressing the fullscreen button the editor used `editor.execCommand('mceFullScreen')`, which caused the insertion point to be positioned incorrectly on iOS.
```

> To resolve this issue, {productname} 6.5, included `skip_focus: true` as a parameter for the `mceFullScreen` command. As a result, the fullscreen button no longer attempts to focus back on the editor.

Reviewer: ```suggestion
To resolve this issue, {productname} 6.5, includes `skip_focus: true` as a parameter for the `mceFullScreen` command. Consequently, the Full Screen button no longer attempts to focus back on the editor.
```
